### ISSUE TITLE
Is GNAT really needed?

### ISSUE THREAD

**Issue by xry111** [glfs-book/glfs#36]
I took a look at Arch and Gentoo, neither lists GNAT (namely gcc-ada in Arch and gnat-gpl in Gentoo) as a dependency of wine.

To me we can try removing it.

**Comment by zeckma-tech** [comment]
I added GNAT as it appeared as a dependency of MinGW-w64-GCC on Arch and thus thought it was also needed for Wine in some way. When we overhauled the MinGW-w64 building process, I learned that only C is required for MinGW-w64, although C++ is good to have too. I forgot that the only reason I added GNAT was because of MinGW-w64, not Wine. So it's probably good to be on the chopping block. Will recompile GCC without Ada support and double check if it's really unnecessary for Wine. It would be nice to get it out of GLFS because having to download a binary to compile something just doesn't feel right to me.
